Fixed nested modules import

spawluk opened the pull request:
When I defined custom database names I got error that `New-SqlConnectionString cmdlet` is not found.

Commit:
Fixed nested modules import

spawluk commented:
My PR unfortunately did not fixed issue with `CommandNotFoundExecption`

Commit:
fixed #38 Command not found when try to create SBFarm

Commit:
Reverted function export.

Commit:
Fixed Missing parameter on auto generated certificate

ryanspletzer commented:
Thanks for taking a crack at it, I've been a little swamped lately... These things are tricky to nail down. Part of me wishes I'd stuck with non class-based resources, but I'm determined to make it work!

spawluk commented:
I can confirm that some parts of it works on Windows Server 2016. 

I'm little surprised of your determination, because even MS stuck to MOF based resources.

Maybe they know that class based resources are hard to debug and maintain? ;)

ryanspletzer commented:
Well from what I understand... it's because they started with the classic approach for certain products so in order to not severely "shake up" those repo's and maintain compatibility they've continued with the same approach. They'd probably have to increment a major version number to start over with a class-based approach for things like Exchange, SharePoint etc... which would be a lot of work + disruption.

I drank the Kool-Aid when I watched Helmick's + Snover's DSC classes on MVA and they were saying class-based was the future, so I went that route. :)

Funny enough, Service Bus for Windows Server and Workflow Manager (which I understand because they're *sort of* deprecated?) and BizTalk Server are the few MSFT products I *haven't* seen DSC resources for from them... Most of the rest of their products have a DSC offering from Microsoft.

ryanspletzer committed:
Merge branch 'dev' into dev

Commit:
Properly importing New-SqlConnectionString
Fixed [Get]SBFarm throwing "You cannot call a method on a null-valued expression"

Commit:
Merge branch 'dev' of github.com:spawluk/ServiceBusForWindowsServerDsc into dev

Commit:
Fixed [Test]SBNamespace. It can use local user groups without domain name

Commit:
Fixed tests

Commit:
Fixed PSSA suggestions

Commit:
Ping CI

Commit:
Fixed tests, updates app veyor config

Commit:
fixed PSSA

Commit:
try install pester from different source

Commit:
Fixed tests finally

spawluk commented:
Finally i managed to fix Set, Get and Test methods. I can tomorrow verify on fresh server if all resources are working as expected

ryanspletzer commented:
Greatly appreciate your efforts!

Commit:
SBFarm GatewayDBConnectionString can be null
DesiredValues[$_] value can be null

Commit:
PowerShell gotcha. Can't use break and continue in script block in ForEach-Object

Commit:
When SBFarm values are not set, it's returned as empty string
